Snail/Observable.swift

```diff
@@ -107,7 +107,7 @@ public class Observable<T>: ObservableType {
     public func flatMap<U>( _ transform: @escaping (T) -> Observable<U>) -> Observable<U> {
         let flatMapped = Observable<U>()
 
-        subscribe(
+        _ = subscribe(
```

Comment:
I think we should add this to the subscribers array right? This way we remove this subscription when `deinit`